src/features/match/service/lookup.ts

```diff
@@ -14,7 +14,7 @@ export type LookUpResponse = {
 
 const fetchMatchList = async () => {
     try {
-        const response = await api.get<LookUpResponse[]>("/api/v1/chat");
+        const response = await api.get<LookUpResponse[]>("/api/v1/chats");
```

Comment:
`src/shared/types` 에 있는 `BaseResponse` 타입을 제네릭으로 사용하면 좋을것 같습니다

```suggestion
        const { data : response } = await api.get<BaseResponse<LookUpResponse[]>>("/api/v1/chats");
```